### DISCUSSION TITLE
作案委托，把报刊亭的狗撸秃毛

### DISCUSSION THREAD

**Discussion by flowingwind1210** [share121/inter-knot#2855] [其他]
![1723992036606](https://github.com/user-attachments/assets/396574b5-0811-4660-bbd6-39499e7cd799)

老子拿到了这个成就，非常不爽，这里求助各位绳匠联合起来，一起疯狂撸嗷呜
![1723992036609](https://github.com/user-attachments/assets/7c977988-2fa6-4641-975a-0636a4f793c8)

**Comment by Kirolei** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/01/18/8481df2cd99d2b00ecb94a92a14c9fe4_7915718047845434249.png" alt="米游兔 OK" width="75" >
禁止欺负狗狗！已上报光映广场动物保护组织缉拿楼主<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可奸笑" width="75" >

**Comment by aizai1173** [comment]
我也得到了<img src="https://img-static.mihoyo.com/communityweb/upload/9939adf214795d97dad9abd90265aab2.png" alt="帕姆_哭哭" width="75" >

**Comment by flowingwind1210** [comment]
话说，我在绳网界面看不见第二张图，各位也是要进网页看吗？<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by I-m-Miss-Lucy-s-dog** [comment]
這委託我接了
